stat_var_renaming/README.md

```diff
@@ -0,0 +1,10 @@
+# DataCommons Statistical Variable Generation Tool
```

Comment:
A couple README requests probably got lost in the 182 comments.

# 1

"The original is here: https://github.com/datacommonsorg/website/blob/4d6f62201337a0603e781a571ca6e09de22a18e2/tools/pv_tree_generator/pop_obs_spec_common.textproto

@IanCostello can you make a comment in the README about how you separated into the two file here?"


# 2

Regarding:
    # (False, True) -> Only group disaster StatVar by populationType
    # if there are more than 1 statistical variables for that group.

"Can you make this a top-level warning in the README?"